docs/getting-started/introduction.md

```diff
@@ -1,121 +1,223 @@
+  use any other service in tandem.
+- **Headless CMS**\
+  Remember: _content_ is just data from a database! Manage any omnichannel digital experience. Deliver file assets and data
+  across websites, apps, kiosks, digital signage... _the sky's the limit!_
+- **Internal Tool Builder**\
+  The whole team can build custom apps! Ditch the spreadsheet and quickly build back-office apps and admin panels for customers,
+  inventory, projects, marketing _or anything else._
+- **Data Management and Analytics**\
+  Establish _a single source of truth_ for all data. Build no-code analytics dashboards to gain insights into company KPIs
+  and other metrics. Coalesce previously siloed department data.
+
+<!--
+mention SaaS in there
+
+:::tip Ready to dive-in?
+
+Get a Project running in minutes. Learn Directus hands-on in the [Quickstart Guide](/getting-started/quickstart/).
+
+:::
+-->
+
+## Why Use It?
+
+**Directus is a simple, data-first solution to complex problems and using it has no downsides.**
+
+Directus was created in 2004, and has been slowly, iteratively improved on for a long time. The core team has carefully
+thought through the app, beginning-to-end, to make sure that using this platform is an all-benefit and no downside
+experience. This platform is plug-and-play, so you're free to link or remove it any time, with no impact on your data.
+You have no vendor lock-in whatsoever, your data is yours. There are no artificial limits on data _(e.g. limits on users
+or roles)_.
+
+<!-- highlight the cost-of-scaling that comes with a platform which charges for users and roles -->
+
+Consider competing platforms in the four general use cases mentioned in the previous section _(BaaS, headless CMS,
+internal tool builder, data management and analytics)_. Due diligence to select the a viable solution one of these
+categories can easily take 6 months to be sure the feature list, pricing, scalability, migration options, etc.. all fit
+the project in question. However, thanks to the plug-and-play nature of Directus, you can link it to the database, test
+it immediately on an existing database _or build a new data model from scratch_, then get rid of it at any time.
+
+The platform is guided by the following core principles:
+
+- **Pure** — No predefined or proprietary schema, with all system metadata stored separately.
+- **Open** — Directus Core is public and open source, with no obfuscated or cloud-only code.
+- **Portable** — Database can be exported or migrated at any time with no vendor lock-in.
+- **Limitless** — No artificial limits or paywalls on users, roles, languages,
+  [Collections](/getting-started/collections), or [Items](getting-started/glossary/items).
+- **Extensible** — Every aspect of the platform is modular to avoid any hard feature ceilings.
+- **Unopinionated** — Choose the stack, database, and architecture as you wish.
+
+<!-- No migration- Directus Cloud?? -->
```

Comment:
@benhaynes you asked me to highlight "no migration" as a thing in "Why" section,  but I felt weird working it in, as we are CTA'ing cloud, which doesn't have BYODB _(doesn't have it yet that is, haha)_.

You want still want me to work this in somehow? Or let it go for the moment?